scripts/chickens.lua

```diff
@@ -0,0 +1,149 @@
+include "constants.lua"
+
+local body,head,tail,lthigh,lknee = piece('body', 'head', 'tail', 'lthigh', 'lknee')
+local lshin,lfoot,rthigh,rknee,rshin = piece('lshin', 'lfoot', 'rthigh', 'rknee', 'rshin')
+local rfoot,rsack,lsack,rblade,lblade = piece('rfoot', 'rsack', 'lsack', 'rblade', 'lblade')
+local mblade,spike1,spike2,spike3 = piece('mblade', 'spike1', 'spike2', 'spike3')
+
+local bMoving   = false
+local SIG_AIM   = 2
+local SIG_AIM_2 = 4
+local SIG_MOVE  = 8
+
+local function Walk()
+	Signal(SIG_MOVE)
+	SetSignalMask(SIG_MOVE)
+	while true do
+		Turn(lthigh, x_axis, math.rad(70), math.rad(115))
+		Turn(lknee, x_axis, math.rad(-40), math.rad(145))
+		Turn(lshin, x_axis, math.rad(20), math.rad(145))
+		Turn(lfoot, x_axis, math.rad(-50), math.rad(210))
+
+		Turn(rthigh, x_axis, math.rad(-20), math.rad(210))
+		Turn(rknee, x_axis, math.rad(-60), math.rad(210))
+		Turn(rshin, x_axis, math.rad(50), math.rad(210))
+		Turn(rfoot, x_axis, math.rad(30), math.rad(210))
+
+		Turn(body, z_axis, math.rad(5), math.rad(20))
+		Turn(lthigh, z_axis, math.rad(-5), math.rad(20))
+		Turn(rthigh, z_axis, math.rad(-5), math.rad(20))
+		Move(body, y_axis, 0.7, 4000)			
```

Comment:
Stick to the original values (also avoid trailing whitespace):
```suggestion
		Move(body, y_axis, 0.7, 11)
```